mev_inspect/compound_liquidations.py

```diff
@@ -0,0 +1,126 @@
+from typing import Dict, List, Optional
+
+from mev_inspect.traces import (
+    get_child_traces,
+    is_child_of_any_address,
+)
+from mev_inspect.schemas.classified_traces import (
+    ClassifiedTrace,
+    DecodedCallTrace,
+    Classification,
+    Protocol,
+)
+
+from mev_inspect.schemas.transfers import ERC20Transfer
+from mev_inspect.schemas.liquidations import Liquidation
+from mev_inspect.classifiers.specs import WETH_ADDRESS
+from mev_inspect.abi import get_raw_abi
+from web3 import Web3
+
+V2_COMPTROLLER_ADDRESS = "0x3d9819210A31b4961b30EF54bE2aeD79B9c9Cd3B"
+V2_C_ETHER = "0x4Ddc2D193948926D02f9B1fE9e1daa0718270ED5"
+
+# helper, only queried once in the beginning (inspect_block)
+def get_all_comp_markets(w3: Web3) -> Dict[str,str]:
```

Comment:
a naming convention I've seen and like is:
- `get_` for in memory (ex: `def get_name(): return self._name`) - most fast
- `find_` for database queries - medium speed
- `fetch_` for network calls - most slow

That helps to see that something like:
```
for i in range(10000):
    ... = get_name()
```
is fine, but that

```
for i in range(10000):
    ... = fetch_name()
```
would be a bottleneck

for that naming then this would be `fetch_all_comp_markets`